site/_docs/history.md

```diff
@@ -56,19 +146,299 @@ other software versions as specified in gradle.properties.
+ `RelBuilder.limit` should apply offset and fetch to previous Sort operator, if possible
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6118">CALCITE-6118</a>]
+ Missing empty `ARRAY` function usage in reference doc
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6121">CALCITE-6121</a>]
+ Invalid unparse for `TIMESTAMP` with SparkSqlDialect
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6109">CALCITE-6109</a>]
+ Linq4j `OptimizeShuttle` should not create new instances of `TernaryExpression` if it does not do any optimization
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6095">CALCITE-6095</a>]
+ Arithmetic expression with `VARBINARY` value causes AssertionFailure
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6150">CALCITE-6150</a>]
+ JDBC adapter for ClickHouse generates incorrect SQL for certain units in the `EXTRACT` function
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6117">CALCITE-6117</a>]
+ Converting `SAFE_CAST` from RexCall to SqlCall fails to add the type as an argument
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6211">CALCITE-6211</a>]
+ `SUBSTRING` with `Integer.MIN_VALUE` as a second parameter raise unexpected exception
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6222">CALCITE-6222</a>]
+ Mysql does not have to_char function
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6213">CALCITE-6213</a>]
+ The default behavior of `NullCollation` in Presto is `LAST`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6227">CALCITE-6227</a>]
+ `ELEMENT`(`NULL`) causes an assertion failure
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6168">CALCITE-6168</a>]
+ `RexExecutor` can throw during compilation
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6118">CALCITE-6118</a>]
+ Missing empty `ARRAY` function usage in reference doc
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5130">CALCITE-5130</a>]
+ `AssertionError`: "Conversion to relational algebra failed to preserve datatypes" when union `VARCHAR` literal and `CAST(null AS INTEGER)`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6180">CALCITE-6180</a>]
+ Append possibility to escape backslash in `LIKE` with `ESCAPE` operator
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6178">CALCITE-6178</a>]
+ `WITH RECURSIVE` query when cloned using `SqlShuttle` looses `RECURSIVE` property
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6332">CALCITE-6332</a>]
+ Optimization `CoreRules.AGGREGATE_EXPAND_DISTINCT_AGGREGATES_TO_JOIN` produces incorrect results for aggregates with groupSets
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6353">CALCITE-6353</a>]
+ Optimization `CoreRules.PROJECT_REDUCE_EXPRESSIONS` crashes while optimizing `ARRAY_CONCAT` expression
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6265">CALCITE-6265</a>]
```

Comment:
`CALCITE-6265` appears two times in the list, one should be removed

Reply:
removed one of the records